mavros/src/plugins/local_position.cpp

```diff
@@ -46,7 +47,8 @@ class LocalPositionPlugin : public MavRosPlugin {
 		lp_nh.param<std::string>("tf/frame_id", tf_frame_id, "local_origin");
 		lp_nh.param<std::string>("tf/child_frame_id", tf_child_frame_id, "fcu");
 
-		local_position = lp_nh.advertise<geometry_msgs::PoseStamped>("local", 10);
+		local_position = lp_nh.advertise<geometry_msgs::PoseStamped>("local/position", 10);
+		local_velocity = lp_nh.advertise<geometry_msgs::TwistStamped>("local/velocity", 10);
```

Comment:
I'm also partial to this topic naming.  As an end user it would be confusing to have local_position/local and local_position/velocity, as "local" really doesn't relate to a physically tangible state.  I'd be happy to update documentation if it meant we could keep this naming (i.e. the ros wiki, PX4 Devguide, PX4 wiki, and github README).

However I know topic renaming can be a nightmare so I would be okay with changing to "local" and "velocity" if otherwise it meant this PR got rejected.